mlflow/tracking/fluent.py

```diff
@@ -363,8 +370,24 @@ def start_run(
         resolved_tags = context_registry.resolve_tags(user_specified_tags)
 
         active_run_obj = client.create_run(
-            experiment_id=exp_id_for_run, tags=resolved_tags, run_name=run_name
+            experiment_id=exp_id_for_run,
+            tags=resolved_tags,
+            run_name=run_name,
         )
+        if log_system_metrics is None:
+            # if `log_system_metrics` is not specified, we will check environment variable.
+            log_system_metrics = MLFLOW_ENABLE_SYSTEM_METRICS_LOGGING.get()
+        if log_system_metrics:
+            try:
+                from mlflow.system_metrics.system_metrics_monitor import SystemMetricsMonitor
+
+                system_monitor = SystemMetricsMonitor(active_run_obj.info.run_id)
+                global run_id_to_system_metrics_monitor
+
+                run_id_to_system_metrics_monitor[active_run_obj.info.run_id] = system_monitor
+                system_monitor.start()
+            except Exception as e:
+                _logger.error(f"Cannot start system metrics monitoring: {e}.")
```

Comment:
```suggestion
                _logger.error(f"Failed to start system metrics monitoring: {e}.")
```

to be consistent with other logging messages.